sys/pm_callouts.c

```diff
@@ -153,7 +153,7 @@ NTSTATUS copyIPv6(const FWPS_INCOMING_VALUES* inFixedValues, FWPS_FIELDS_OUTBOUN
     return STATUS_SUCCESS;
 }
 
-void redir_from_callout(FWPS_CLASSIFY_OUT* classifyOut, pportmaster_packet_info packetInfo, pportmaster_packet_info redirInfo, PNET_BUFFER nb, size_t ipHeaderSize, BOOL dns) {
+void redir_from_callout(pportmaster_packet_info packetInfo, pportmaster_packet_info redirInfo, PNET_BUFFER nb, size_t ipHeaderSize, BOOL dns) {
     void* packet;
```

Comment:
I'd either use `portmaster_packet_info*` or stick to the Windows convention of using all-uppercase `PPORTMASTER_PACKET_INFO`. The second lowercase `p` is easy to overlook as once is not expecting it to be there.